pkgs/development/compilers/gcc/common/configure-flags.nix

```diff
@@ -23,6 +23,7 @@
 , langObjC
 , langObjCpp
 , langJit
+, enableBootstrap ? stdenv.targetPlatform == stdenv.hostPlatform
```

Comment:
I think `disableBootstrap ? targetPlatform != hostPlatform` is a more natural name: setting `enableBootstrap = true` will not force the bootstrap for cross-compilers.

Reply:
Good point.  Fixed in c29e775359eead2d0a1fe0e2c7eacf53087d64aa.